client/src/page/IntroPage.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import OutlinedContainer from "../components/OutlinedContainer";
+import styled from "@emotion/styled";
+import { PageWrapper } from "../components/PageWrapper";
+import IconArrowLeftSrc from "../assets/IconArrowLeft.svg";
+import IconArrowRightSrc from "../assets/IconArrowRight.svg";
+import IconSpraceSrc from "../assets/IconSpace.svg";
+import ProceedButton from "../components/ProceedButton";
+
+function IntroPage() {
+  return (
+    <IntroPageWrapper>
+      <OutlinedContainer>
+        <IntroTextbox>
+          <span>Please help!</span>
+          <p>
+            The little astronaut is stranded on a lonley planet. He needs to get
+            back to his spaceship as soon as possible.
```

Comment:
This poor astronaut...
![image](https://user-images.githubusercontent.com/68224104/94689193-de6d0e00-032e-11eb-8fea-0c7ebc8dc4fd.png)